core/consensus/aura/src/lib.rs

```diff
@@ -615,7 +615,7 @@ impl<B: Block, C, E> Verifier<B> for AuraVerifier<C, E> where
 }
 
 /// The Aura import queue type.
-pub type AuraImportQueue<B, C, E> = BasicQueue<B, AuraVerifier<C, E>>;
+pub type AuraImportQueue<B> = BasicQueue<B>;
```

Comment:
Maybe we can remove this type, since the `BasicQueue` type is now an opaque interface that will be used by everyone (regardless of underlying verifier).

Reply:
The problem with this is that the `AuraImportQueue` is exported and used outside of the crate, while `BasicQueue` is private(and probably should be). Perhaps `AuraImportQueue` can be replaced by a `ImportQueue`, however I think that's going to require boxing it 😄 , and will also require some additional work since I would be very surprised if it didn't require changing some of the code using it. Perhaps we can look into this as a follow-up?

Reply:
Didn't think about that. Sure that's reasonable. :+1: